unity/Assets/Scripts/MessagePack/ExtensionHeader.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) All contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
Can we include a message pack distribution in manifest.json? instead of including all these source files .

Reply:
I believe the copyright header is a requirement from legal when we open sourced the project.

Reply:
I meant including the binary instead of the source, discussed through slack and resolved